Bots/DotNet/Consumers/Composer/SimpleComposerHostBot/runtime/customaction/CustomActionComponentRegistration.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.Bot.Builder;
```

Comment:
Here the copyright header is missing.

Reply:
Opened an issue with composer reporting this (https://github.com/microsoft/BotFramework-Composer/issues/4923)